impeller/renderer/backend/metal/context_mtl.mm

```diff
@@ -71,18 +71,31 @@ static bool DeviceSupportsComputeSubgroups(id<MTLDevice> device) {
     id<MTLDevice> device,
     id<MTLCommandQueue> command_queue,
     NSArray<id<MTLLibrary>>* shader_libraries,
-    std::shared_ptr<fml::ConcurrentTaskRunner> worker_task_runner,
     std::shared_ptr<const fml::SyncSwitch> is_gpu_disabled_sync_switch)
     : device_(device),
       command_queue_(command_queue),
-      worker_task_runner_(std::move(worker_task_runner)),
       is_gpu_disabled_sync_switch_(std::move(is_gpu_disabled_sync_switch)) {
   // Validate device.
   if (!device_) {
     VALIDATION_LOG << "Could not setup valid Metal device.";
     return;
   }
 
+  // Worker task runner.
+  {
+    raster_message_loop_ = fml::ConcurrentMessageLoop::Create(4u);
+    raster_message_loop_->PostTaskToAllWorkers([]() {
+      [[NSThread currentThread] setThreadPriority:1.0];
+      sched_param param;
+      int policy;
+      pthread_t thread = pthread_self();
+      if (!pthread_getschedparam(thread, &policy, &param)) {
+        param.sched_priority = 50;
+        pthread_setschedparam(thread, policy, &param);
```

Comment:
Are both NSThread and pthread priority calls necessary? I expected only the first on to be a thing on iOS.

Reply:
This reminds me that we should probably use libDispatch with the right QoS classes on iOS. The platform will create the right number and nature of workers then. We had prototyped this on iOS in [pthread_setschedparam ](https://github.com/flutter/engine/pull/35355/files) but backed it out. Perhaps we can resuscitate it.